histogram_file_manager/models.py

```diff
@@ -3,6 +3,38 @@
 
 # from django.conf import settings
 
+class HistogramDataFileContents(models.Model):
+    GRANULARITY_UNKNOWN = "unk"
+    GRANULARITY_RUN = "run"
+    GRANULARITY_LUMISECTION = "lum"
+    DIMENSIONALITY_UNKNOWN = 0
+    DIMENSIONALITY_1D = 1
+    DIMENSIONALITY_2D = 2
+
+    HISTOGRAM_DIMENSIONS_CHOICES = (
+        (DIMENSIONALITY_UNKNOWN, "Unknown"),
+        (DIMENSIONALITY_1D, "1D"),
+        (DIMENSIONALITY_2D, "2D"),
+    )
+
+    DATAFILE_GRANULARITY_CHOICES = (
+        (GRANULARITY_UNKNOWN, "Unknown"),
+        (GRANULARITY_RUN, "Run"),
+        (GRANULARITY_LUMISECTION, "Lumisection"),
+    )
+
+    data_dimensionality = models.PositiveIntegerField(
+        default = DIMENSIONALITY_UNKNOWN, 
+        choices = HISTOGRAM_DIMENSIONS_CHOICES, 
+        blank   = True
+    )
+
+    granularity = models.CharField(
+        max_length = 3,
+        choices    = DATAFILE_GRANULARITY_CHOICES,
+        default    = GRANULARITY_UNKNOWN,
```

Comment:
As above, make it `GRANULARITY_RUN` for now.

Reply:
Done.